help/deploy/complete-setup/configure-auth.md

```diff
@@ -3,75 +3,94 @@ title: "Configure authentication"
 sidebar_position: 1
 ---
 
-Out-of-the-box, Datagrok offers many authentication methods, including popular methods such as SSO and OAuth.
+Datagrok supports authentication using:
 
-To accommodate any enterprise needs, we can develop a customer-specific integration.
+* [Login-password](#login-password-authentication)
+* [Oauth](#oauth-authentication): 
+  * [Google](#google-authentication)
+  * [Facebook](#facebook-authentication)
+  * [GitHub](#github-authentication)
+* [OpenID](#openid-authentication)
+* [LDAP](#ldap-authentication)
 
-All methods can be enabled separately or combined.
+You can enable all authentication methods separately or combined.
 
-[Learn more about authentification](../govern/authentication.md).
+If supported authentication methods do not work for you, contact us on <info@datagrok.ai> or [our community](https://community.datagrok.ai/) and we will discuss options for your specific case.
 
-## Common options
+[Learn more about authentification](../../govern/authentication.md).
 
-If you want to display something on the login window, set the 'Prompt' option
-![Login prompt](login-prompt.png)
+## Common options
 
-Enable the 'Send Welcome Email' option to send welcome emails to new signed-up users.
+You can display a text on the login window, like greetings words etc.
+To do so set the *Prompt* option in **Settings** > **Users and Sessions** > **General**:
+<p align="center">
+  <img alt="login-prompt" src={require('./login-prompt.png').default} width="300px"/>
+</p>
 
 ## Login-password authentication
 
-It is the most basic method to authenticate users with Datagrok.
-
 To configure login-password authentication for Datagrok:
 
-1. Go to the Datagrok Settings section 'Users and Sessions'; this section contains all authentication settings.
-2. Enable 'Internal authentication' to use the login-password method
-3. Signup option is alterable; you can disable it using the 'Signup Allowed' option
-4. To restrict from which domains people can sign up to the platform, use the 'Signup Domains Whitelist' option. You can
+1. On the left **Sidebar** go to **Settings** > **Users and Sessions**. Authentication settings page appears.
+2. Enable *Internal authentication* to use the login-password method.
+3. Adjust the *Signup* option. You can disable or enable it using the *Signup Allowed* and *Allow Oauth Signup* checkboxes.
+4. For better security you can restrict from which domains people can sign up to the platform. To do so use the *Signup Domains Whitelist* option. You can
```

Comment:
'To do so' is a clear sign that the sentence needs to be rewritten.
For example,
```suggestion
4. To restrict domains people can sign up from, use the *Signup Domains Whitelist* option. Separate several domains with commas.
```